docs/omero_figure_scripting.rst

```diff
@@ -13,44 +13,43 @@ Setup
 
 -  Install the OMERO.figure web app as described at https://pypi.org/project/omero-figure/
 -  Upload the script :download:`Split_View_Figure.py <../scripts/Split_View_Figure.py>` to the OMERO scripting service
+-  For the `Shapes heatmap from OMERO.table` example, you'll need to draw ROIs on an Image and create an OMERO.table on the
```

Comment:
A link to the subchapter could be added similarly to https://omero-guides.readthedocs.io/en/latest/upload/docs/metadata-ui.html#populatemetadatapdi (you will need an anchor just in front of the subchapter, see
![Screenshot 2021-06-01 at 14 43 50](https://user-images.githubusercontent.com/2478303/120333679-d7e0cd00-c2e7-11eb-90df-ab499a946ec3.png)


 in https://raw.githubusercontent.com/ome/omero-guide-upload/master/docs/metadata-ui.rst